src/libopensc/iso7816.c

```diff
@@ -1201,6 +1201,9 @@ iso7816_build_pin_apdu(struct sc_card *card, struct sc_apdu *apdu,
 	switch (data->cmd) {
 	case SC_PIN_CMD_VERIFY:
 		ins = 0x20;
+		/* detect overloaded APDU with SC_PIN_CMD_GET_INFO */
+		if (data->pin1.len == 0)
```

Comment:
what about pinpads? I think they still go through this code path in the iso7816 code.

Reply:
True. I've added a commit to reflect this.